docs/src/basics/structs_tuples_and_enums.md

```diff
@@ -103,34 +120,56 @@ enum InventoryEvent {
 }
 ```
 
+If we wanted to instantiate an enum with some interior data, it looks like this:
+
 ```sway
-enum Color {
-    Blue: (),
-    Green: (),
-    Red: (),
-    Silver: (),
-    Grey: (),
+fn main() {
+  let event = InventoryEvent::ItemLoss(Claim {
```

Comment:
Weird indentation. While here, can you move these to `examples/`?

Reply:
By "move" do you mean add the same examples into contract form under `examples/` or to completely remove everything to do with enums and move that under `examples/`?

Reply:
Completely move all inline code in this page into the examples directory, and include it with `{{#include ...}}`.